### ISSUE TITLE
No ANSI color codes output option?

### ISSUE THREAD

**Issue by sentientnebula** [mantoni/mochify.js#78]
I'm running mochify in a small pane (buffer) inside Sublime Text 3.  It works great except for ST not supporting ANSI codes.  I get a lot of extra gibberish instead of the green/red highlights which makes it hard to read.  Is there a simple way to get straight text output?

**Comment by sentientnebula** [comment]
I thought 
mochify -R mocha-unfunk-reporter
might be a work-around, so I installed that NPM, but get an invalid reporter error when trying to use it.

I looks like the taps reporter uses no ANSI though--although output format is totally different.

**Comment by mantoni** [comment]
Color is enabled or disabled based on [supports-color](https://www.npmjs.com/package/supports-color). According to the sources you can enforce no-color by setting an environment variable `TERM = 'dumb'`.

I would support a command line option to force-enable / force-disable color. If you care to have a look, the [color config for Mocha is set in mocaccino](https://github.com/mantoni/mocaccino.js/blob/master/lib/mocaccino.js#L63).

Mochify could support the same options as Mocha:

```
-c, --colors        force enabling of colors
-C, --no-colors     force disabling of colors
```

**Comment by mantoni** [comment]
I came across [a bug in mocaccino]() today. The result of the color detection was never applied correctly. It's fixed now and should work after a fresh Mochify install.

There is now also a new Mochify version (`v2.7.0`) with support for `--color` and `--no-color`. This should get you sorted in case the detection still fails for you.

**Comment by sentientnebula** [comment]
Thanks.  That did the trick!  After updating to v2.7.0 everything works perfectly from inside a Sublime Text buffer.  I don't even have to use the --no-color color option now that color detection is being applied.

Adding the --no-color option is a very good idea though.  I looked over the supports-color code and found some problems with the logic that cause it to return 0 when it shouldn't.

I tested inside a bash shell in an an xterm, and had a difficult time getting it to return 1.

```
export TERM='dumb'
supports-color
echo $?
0

export COLORTERM=0
supports-color
echo $?
0

unset COLORTERM
supports-color
echo $?
1
```

For some odd reason, supports-color forces color support on if the COLORTERM variable exists regardless of whether it is set to true or false and returns before even checking TERM.

After finally getting a return of 1, I ran mochify and was suprised to see it was still outputting color.  But you fixed that right up :)  Thanks a lot for the quick response.  Having in little box inside my editor displaying real-time test results is most excellent.   mochify is very satisfying little tool!
